website/static/twitter/credentials.py

```diff
@@ -0,0 +1,14 @@
+
+
+# Create variables for each key, secret, token
```

Comment:
I think these .py files in the `/static/` were added by mistake. They should removed. We should assume that these credentials have been compromised and rotate them.